docker-compose.yaml

```diff
@@ -10,7 +10,8 @@ services:
       - NEXT_PUBLIC_API_BASE_URL_QUERY_SERVER=${NEXT_PUBLIC_API_BASE_URL_QUERY_SERVER:-http://query:3004/queries/v1}
       - NEXT_PUBLIC_API_BASE_URL_QUERY_CLIENT=${NEXT_PUBLIC_API_BASE_URL_QUERY_CLIENT:-http://localhost:3004/queries/v1}
       - NEXT_PUBLIC_API_BASE_URL_COMMAND=${NEXT_PUBLIC_API_BASE_URL_COMMAND:-http://localhost:3002/commands/v1}
-      - NEXT_PUBLIC_API_BASE_URL_AUTH=${NEXT_PUBLIC_API_BASE_URL_AUTH:-http://localhost:3001}
+      - NEXT_PUBLIC_API_BASE_URL_AUTH_SERVER=${NEXT_PUBLIC_API_BASE_URL_AUTH_SERVER:-http://localhost:3001}
```

Comment:
This should point to auth( I will fix it just writing it down so I don't forget it)